src/Helpers.elm

```diff
@@ -0,0 +1,55 @@
+module Helpers exposing (..)
+
+{-| Returns the n first digits after the comma in a float
+
+    >>> nDigits 2 123.45
+    "45"
+
+    >>> nDigits 0 125
+    ""
+
+    >>> nDigits 1 1.99
+    "0"
+
+    >>> nDigits 2 1.0
+    "00"
+
+    >>> nDigits 2 -1.0001
+    "00"
+
+    >>> nDigits 2 0.01
+    "01"
+
+    >>> nDigits 2 0.10
+    "10"
+-}
+
+
+nDigits : Int -> Float -> String
+nDigits digits f =
+    let
+        multiplicator =
+            toFloat (10 ^ digits)
+
+        fint =
+            (round (f * multiplicator))
+    in
+        splitIntRec fint []
+            |> String.concat
+            |> String.right digits
+            |> String.padLeft digits '0'
+
+
+{-| Recursive helper to format an integer
+
+    >>> splitIntRec 12345 []
+    ["12", "345"]
+-}
+splitIntRec : Int -> List String -> List String
```

Comment:
>  It is not exposed to users.

I'm not takingo into account final users. Contributors (like future-me and future-you) might also struggle with internal APIs. I'll try to refactor it and add here as a suggestion later today ; )